cmake/CPM.cmake

```diff
@@ -5,7 +5,7 @@
 # MIT License
 # -----------
 #[[
-  Copyright (c) 2021 Lars Melchior and additional contributors
+  Copyright (c) 2021 Lars Melchior, NVIDIA CORPORATION, and additional contributors
```

Comment:
As for your previous PR, I don't think adding individual contributors here is a good idea.

I will work on adding a separate list of contributors in the near future